trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/loss-given-default.njk

```diff
@@ -5,6 +5,8 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle %}What’s the loss given default (LGD) for {{ deal.submissionDetails.supplierName}}{% endblock %}
```

Comment:
Should we have a question mark at the end of the title?

Reply:
I was not sure about question marks in titles, because we add `, Trade Finance Manager, gov.uk` to all of them, but I added `?` for consistency.